components/eam/cime_config/config_compsets.xml

```diff
@@ -61,16 +61,32 @@
     <lname>2010_EAM%CMIP6_ELM%SPBC_MPASSI%PRES_DOCN%DOM_MOSART_SGLC_SWAV</lname>
   </compset>
 
+  <!-- *********************************** -->
+  <!-- Single column model -->
+  <!-- *********************************** -->
+
   <compset>
-    <alias>FSCM5A97</alias>
-    <lname>AR97_EAM%SCAM_ELM%SP_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV</lname>
+    <alias>FSCM-ARM97</alias>
+    <lname>AR97_EAM%SCM_ELM%SP_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV</lname>
```

Comment:
For clarity, we can also change AR97 to ARM97 in the long name. Similarly for FSCM-ARM97-MMF1.